docs/release-notes/5.7.44-49.md

```diff
@@ -0,0 +1,88 @@
+
+    We recommend that you either [upgrade to MySQL 8.0 or stay on 5.7; we'll support you].
+
+
+## Release Information
+
+Percona Server for MySQL 5.7.44-49 contains the fix for CVE-2024-20963 and a portability fix.
+
+## Bug fixes
+
+[PS-9096] Fixes the following:
+
+  * [Bug#35764496]: Running too much data through MD5() causes the server to stop. The cause was accessing invalid memory or over-allocating while computing hashes in the md5 encryption methods.
+
+  * [Bug#35967676]: Fixes a compilation failure fix in VS2022.
+
+## CVE and Third-party library information
+
+!!! admonition "important"
+
+    The binary tarballs are not shipped with the updated third-party libraries.
+
+
+The following Common Vulnerabilities and Exposures (CVE) have been identified in the third-party libraries:
+
+  * [CVE-2023-38545]
+
+  * [CVE-2023-5363]
+
+Please [upgrade the affected third-party library to the latest version](#instructions-to-upgrade-third-party-libraries), at least curl v8.4.0 and OpenSSL v3.0.12. Also, the operating systems may port these fixes into the default versions of the libraries installed in the systems. For example, the OpenSSL CVE is fixed in the patched version on the following platforms:
+
+  * [Ubuntu 22.04 LTS]
+
+  * [Debian 12]
+
+### Instructions to upgrade third-party libraries
+
+The following are basic instructions on updating these libraries based on the platforms. Your environment may be different.
+
+??? note "Distributions using the APT package manager"
+
+    The following instructions update the OpenSSL library.
+    {.power-number}
+
+    1. Stop the server.
+
+    2. Get the update
+
+        ```{.bash data-prompt="$"}
+        $ sudo apt update; sudo apt install libssl-dev openssl;
+        ```
+
+    3. Restart the server
+
+
+## Useful links
+
+[Install Percona Server for MySQL](https://www.percona.com/doc/percona-server/5.7/installation.html)
+
+The [Percona Server for MySQL GitHub repository](https://github.com/percona/percona-server)
+
+[Contribute to the documentation](https://github.com/percona/psmysql-docs/blob/8.0/contributing.md)
+
+Download product binaries, packages, and tarballs at [Percona Product Downloads](https://www.percona.com/downloads)
+
+For [training](https://www.percona.com/training), contact [Percona Training - Start learning now](https://learn.percona.com/contact-me)
+
+
+[upgrade to MySQL 8.0 or stay on 5.7; we'll support you]: https://www.percona.com/navigating-mysql-5-7-end-of-life
+
+[PS-9096]: https://perconadev.atlassian.net/browse/PS-9096
+
+[Bug#35764496]: https://github.com/mysql/mysql-server/commit/fc8b390e7bf
+
+[Bug#35967676]: https://github.com/mysql/mysql-server/commit/4611cc258e3
+
+[Ubuntu 22.04 LTS]: https://launchpad.net/ubuntu/+source/openssl/3.0.2-0ubuntu1.12
+
+[Debian 12]: https://lists.debian.org/debian-security-announce/2023/msg00228.html
+
+[CVE-2023-38545]: https://curl.se/docs/CVE-2023-38545.html
+
+[CVE-2023-5363]: openssl.org/CVE-2023-5363
```

Comment:
The url for CVE-2023-5363 should be https://www.openssl.org/news/vulnerabilities.html#CVE-2023-5363.